### PR TITLE
fix(documentation): Updated readme to reflect that GraphiQL is not available anymore

### PR DESCRIPTION
**Pull request by StephanH90**
Also added set the DEV flag to true in the example `docker-compose.yml` file, which is something that I struggled with at the start.

However, that change is perhaps controversial.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ services:
     #   - ./data_sources.py:/app/caluma/extensions/data_sources.py
     #   - ./format_validators.py:/app/caluma/extensions/format_validators.py
     environment:
+      - ENV=dev
```
Comment (czosel) [inline docker-compose.yml:27]: Should we add a comment explaining that this should be removed in production?

Comment (open-dynaMIX) [inline docker-compose.yml:27]: This is already set [in the docker-compose.override.yml#L26](https://github.com/projectcaluma/caluma/blob/master/docker-compose.override.yml#L26). Value can be anything except `production`.

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Otherwise LGTM :100:

File: docker-compose.yml (hunk 1 above)
Comment (czosel) [reply docker-compose.yml:27]: Ah, right! @StephanH90 has the override file not been picked up in your case?

Comment (StephanH90) [reply docker-compose.yml:27]: In that case I'm not sure why that didn't work. I remember that when I made a fresh install of docker and a clone of the repo I got a bunch of warning along the lines of: "secret key needs to be set", etc.

However, I suspect that this was a problem with my local setup.

**Review by open-dynaMIX — APPROVED** [review]

**Pull request merged**